script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
**[REQUIRED CHANGES]**

- Please, always add an end of a new line in all your file as it's considered best practice in writing javascript code. The official definition states that a new line is a sequence of zero or more non-newline characters with a terminating newline character. Therefore, to eliminate any confusion, we should add a new line so that text editors and other programs can tell that the end of the file actually consists of a line.
_**Please, kindly read more [about new line](https://medium.com/swlh/javascript-code-styling-best-practices-this-lines-and-functions-b354e70e1e67#:~:text=Newline%20should%20be%20added%20at,the%20end%20of%20the%20line.)**_